solutions/fully-configurable/variables.tf

```diff
@@ -243,7 +243,7 @@ variable "provision_trusted_profile_projects" {
 
 variable "trusted_profile_name" {
   type        = string
-  description = "The name of the trusted profile. Required if `provision_trusted_profile_projects` is true."
```

Comment:
why did you remove the last sentence? This is still valid right?

Reply:
fixed